tagstudio/src/qt/ts_qt.py

```diff
@@ -841,6 +869,20 @@ def run_macro(self, name: str, entry_id: int):
                             mode="replace",
                         )
 
+    def copy_entry_fields_action_callback(self):
+        for item in self.selected.__reversed__():
+            if item[0] == ItemType.ENTRY:
+                entry = self.lib.get_entry(item[1])
```

Comment:
- `__reversed__` is a private method, so ideally we shouldnt use it
- `item` can be unpacked directly into `item_type` and `item_id`, so it's more obvious what values we're working with

```suggestion
        for item_type, item_id in reversed(self.selected):
            if item_type == ItemType.ENTRY:
                entry = self.lib.get_entry(item_id)
```